File: src/main/java/frc/robot/utils/LocationSensing.java
package frc.robot.utils;

import java.util.List;
import java.util.Queue;
import java.lang.Math;

public class LocationSensing {


    private double distance;

    private double angle;

    private double xpos;
    private double ypos;
    private double corrx;
    private double corry;

   

    public double[] getPosition(double area,double angle) {
        distance = Math.sqrt(area);
        xpos = distance*(Math.sin(-angle));
        ypos = distance*(Math.cos(-angle));
        corrx= 0.85*(-Math.cos(-angle)) + xpos;
        corry= 0.85*(-Math.sin(-angle)) + ypos;
        return new double[] {0.0,0.0};
      }

}

File: src/main/java/frc/robot/utils/LocationSensing.kt
package frc.robot.utils

class LocationSensing {
    private var distance = 0.0
    private val angle = 0.0
    private var xpos = 0.0
    private var ypos = 0.0
    private var corrx = 0.0
    private var corry = 0.0
    fun getPosition(area: Double, angle: Double): DoubleArray {
        distance = Math.sqrt(area)
        xpos = distance * Math.sin(-angle)
        ypos = distance * Math.cos(-angle)
        corrx = 0.85 * -Math.cos(-angle) + xpos
        corry = 0.85 * -Math.sin(-angle) + ypos
        return doubleArrayOf(0.0, 0.0)
    }
}
